radicle-cli/src/git.rs

```diff
@@ -175,24 +175,27 @@ pub fn is_signing_configured(repo: &Path) -> Result<bool, anyhow::Error> {
 }
 
 /// Return the list of radicle remotes for the given repository.
-pub fn remotes(repo: &git2::Repository) -> anyhow::Result<Vec<(String, NodeId)>> {
-    let mut remotes = Vec::new();
-
-    for name in repo.remotes().iter().flatten().flatten() {
-        let remote = repo.find_remote(name)?;
-        for refspec in remote.refspecs() {
-            if refspec.direction() != git2::Direction::Fetch {
-                continue;
-            }
-            if let Some((peer, _)) = refspec.src().and_then(self::parse_remote) {
-                remotes.push((name.to_owned(), peer));
-            }
-        }
-    }
-
+pub fn rad_remotes(repo: &git2::Repository) -> anyhow::Result<Vec<git2::Remote>> {
+    let remotes: Vec<_> = repo
+        .remotes()?
+        .iter()
+        .filter_map(|name| {
+            let remote = repo.find_remote(name?).ok()?;
+            let url = remote.url()?;
+            url.starts_with(radicle::storage::git::transport::local::Url::SCHEME)
+                .then_some(remote)
+        })
+        .collect();
     Ok(remotes)
 }
 
+/// Return true is the remote is stored inside the  the list of radicle remotes.
+pub fn is_remote(repo: &git2::Repository, alias: &str) -> anyhow::Result<bool> {
```

Comment:
```suggestion
/// Check if the git remote is configured for the `Repository`.
pub fn has_remote(repo: &git2::Repository, alias: &str) -> anyhow::Result<bool> {
```

Reply:
Maybe comment not needed after rename -> has_remote

Reply:
`has_remote` this will restore the previous solution that goes in conflict with @cloudhead 

Naming is hard 😸